homework/12_A/10_Zlatina_Lilova/homework02/chess.py

```diff
@@ -0,0 +1,87 @@
+from dataclasses import dataclass
+import termcolor as tc
+
+
+@dataclass
+class WhiteFigures:
+    pawn = " ♙  "
+    rook = ' ♖  '
+    knight = ' ♘  '
+    bishop = ' ♗  '
+    king = ' ♔  '
+    queen = ' ♕  '
+    order_of_last_row = ['rook', 'knight', 'bishop',
+                         'queen', 'king', 'bishop', 'knight', 'rook']
+
+
+@dataclass
+class BlackFigures:
```

Comment:
This class should have been an Enum. The fact that you just enumerate values and never instantiate the class is a give-away.